pkg/controllers/binding/common.go

```diff
@@ -118,6 +103,28 @@ func ensureWork(
 	return nil
 }
 
+func extractBingCoreParameters(binding metav1.Object, scope apiextensionsv1.ResourceScope) *bindingCoreParameters {
```

Comment:
a typo `Bing`.
In addition, how about renaming struct to `ensureWorkParameters`